src/SourceBuild/content/eng/pipelines/ci.yml

```diff
@@ -39,7 +39,7 @@ variables:
   value: ${{ eq(variables['Build.Reason'], 'PullRequest') }}
 
 stages:
-- ${{ if not(eq(variables.isPRTrigger, 'true')) }}:
+- ${{ if and(not(eq(variables.isPRTrigger, 'true')), eq(variables['System.TeamProject'], 'internal')) }}:

```

Comment:
Hmm, isn't `not(eq(` the same as `ne`? Would you mind simplifying the condition?

Reply:
Fixed - there was one more occurrence in this file that I fixed as well.